src/components/Toolbar.ts

```diff
@@ -0,0 +1,86 @@
+import { createElement } from "react";
+import * as classNames from "classnames";
+import * as Toolbar from "react-big-calendar/lib/Toolbar";
+export default class CustomToolbar extends Toolbar {
+
+    render() {
+        const leftButton = this.props.customViews.filter((customView: any) => customView.position === "left");
+        const centerButton = this.props.customViews.filter((customView: any) => customView.position === "center");
+        const rightButton = this.props.customViews.filter((customView: any) => customView.position === "right");
+
+        if (this.props.viewOption === "standard") {
+            const standardViewButtons = [ "day", "week", "month" ];
+
+            return createElement("div", { className: classNames("calendar-toolbar") },
```

Comment:
don't create a rendering fro the standard, create a standard config, that has the same output